lib/growing_file.js

```diff
@@ -49,6 +49,7 @@ GrowingFile.open = function(path, options) {
 GrowingFile.prototype.destroy = function() {
   this.readable = false;
   this._stream = null;
+  this._ended = true;
```

Comment:
Actually I'm wondering if it's more appropriate to override `GrowingFile.prototype.end` or set `this._idleTimeout` to `this._timeout + 1` and then call `this.end();` interally.